toolbox

```diff
@@ -946,14 +946,23 @@ enter()
     create_toolbox_container=false
     prompt_for_create=true
     shell_to_exec=/bin/bash
+    run "PS1=\"$toolbox_prompt\" $shell_to_exec" "$SHELL" "--interactive --tty"
```

Comment:
The `PS1` is now set by the `profile.d/toolbox.sh` start-up script, and hence this is simpler now. See #148 

`--interactive` and `--tty` are still necessary in `run()` for things like `emacs -nw`.